app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -181,13 +181,36 @@ def test_data_fetch_in_loop(self, lifecycle, mock_delay):
     @patch("experimenter.jetstream.tasks.fetch_experiment_data.delay")
     def test_data_expired_in_loop(self, lifecycle, mock_delay):
         experiment = NimbusExperimentFactory.create_with_lifecycle(lifecycle)
+        experiment.results_data = {
+            "daily": None,
+            "metadata": None,
+            "overall": None,
+            "show_analysis": False,
+            "weekly": None,
+        }
+        experiment.save()
         experiment.changes.all().filter(
             old_status=NimbusExperiment.Status.LIVE,
             new_status=NimbusExperiment.Status.COMPLETE,
         ).update(changed_on=datetime.date.today() - datetime.timedelta(days=4))
         tasks.fetch_jetstream_data()
         mock_delay.assert_not_called()
 
+    @parameterized.expand(
+        [
+            (NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE,),
+        ]
+    )
```

Comment:
Naive question.. is it worth writing this in the parameterized format if we're only testing one lifecycle? Or do we plan to add more later?

Reply:
Fair question, I think it's not necessary but it does make it easier to add more later.